MetaMask/metamask-extension
# Swaps: Improve hardware wallet UX

## Explanation
User story: As a user, I want to understand why two separate signatures are required when using a hardware wallet

We are adding 2 new pages:
- For 2 confirmations on a HW wallet
- For 1 confirmation on a HW wallet

## Manual testing steps
Test case: "1 confirmation: ETH -> ERC20 with a HW wallet"
1) Switch to a HW wallet
2) Click on "Swap" on the main page
3) Select "Token From" as ETH and "Token To" as UNI (or any other ERC20 token)
4) Click on "Review Swap"
5) Click on "Swap" on the View Quote page
6) See the Awaiting Signatures page for 1 confirmation

To see the page for 2 confirmations you can do either an ERC20 -> ETH swap or ERC20 -> ERC20 swap.

## Screenshots
A new page for 1 confirmation on a HW wallet:
![image](https://user-images.githubusercontent.com/80175477/117189224-c95bd000-ad92-11eb-94bf-7822d59e2fa4.png)

A new page for 2 confirmations on a HW wallet:
![image](https://user-images.githubusercontent.com/80175477/117189284-ded0fa00-ad92-11eb-9bba-512c1f034345.png)
